core/templates/services/audio-bar-status.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for AudioBarStatusService.
+ */
+
+import { TestBed } from '@angular/core/testing';
+
+import { AudioBarStatusService } from 'services/audio-bar-status.service';
+
+describe('AudioBarStatusService', () => {
+  let audioBarStatusService: AudioBarStatusService = null;
+
+  beforeEach(() => {
+    audioBarStatusService = TestBed.get(AudioBarStatusService);
+  });
+  // This is because by default the audiobar is collapsed
+  it('should be collapsed by default', () => {
+    expect(audioBarStatusService.isAudioBarExpanded()).toBe(false);
+  });
+
+  it('should return true if AudioBar is expanded', () => {
+    audioBarStatusService.expandAudioBar();
```

Comment:
Add a baseline verification i.e initially it was false for `isAudioBarExpanded`.

Reply:
It's the above it block, right?